index.html

```diff
@@ -0,0 +1,137 @@
+        <div class="date-venue">
+          <p class="date">2022-03-21(MON) ~ 25(FRI)</p>
+          <p class="venue">@ Moscone Center San Francisco, CA USA</p>
+        </div>
+      </div>
+    </header>
+    <!-- Program section -->
+    <section class="program">
+      <h2 class="titles">Main Program</h2>
+      <div class="details">
+        <!-- program 1 -->
+        <div class="detail-col">
+          <img src="/assets/pro1.png" alt="detail1">
+          <h2>Lecture</h2>
+          <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+        </div>
+        <!-- program 2 -->
+        <div class="detail-col">
+          <img src="/assets/pro2.png" alt="detail2">
+          <h2>GDC Exhibition</h2>
+          <p>Appreciate various creations applying GDC license of artists, organized from GDC home office.</p>
+        </div>
+        <!-- program 3 -->
+        <div class="detail-col">
+          <img src="/assets/pro3.png" alt="detail3">
+          <h2>Forum</h2>
+          <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <!-- program 4 -->
+        <div class="detail-col">
+          <img src="/assets/pro4.png" alt="detail4">
+          <h2>Workshop</h2>
+          <p>Try creating your own work using open source license rather than just watching at it.</p>
+        </div>
+        <!-- program 5 -->
+        <div class="detail-col">
+          <img src="/assets/pro5.png" alt="detail5">
+          <h2>GDC Ignite</h2>
+          <p>Get opportunities to network with GDC affiliates around the world, also after the conference.</p>
+        </div>
+      </div>
+      <a href="#" class="desktop-text">see the whole program</a>
+      <a href="#" class="mobile-text">Join GDC 2022</a>
+    </section>
+    <!-- featured speakers section -->
+    <section class="speakers">
+      <h2 class="titles">Featured Speakers</h2>
+      <div class="speakers-list"></div>
+      <button type="button" class="btn-extend">
+        <span class="btn-text">More</span>
+        <img src="/assets/arrow_down.png" class="btn-pic" alt="down">
+      </button>
+    </section>
+    <!-- partners section -->
+    <section class="partners">
+      <h2 class="titles">Partners</h2>
+      <div class="list">
+        <img src="/assets/partners/partner1.png" alt="partners">
+        <img src="/assets/partners/partner2.png" alt="partners">
+        <img src="/assets/partners/partner3.png" alt="partners">
+        <img src="/assets/partners/partner4.png" alt="partners">
+        <img src="/assets/partners/partner5.png" alt="partners">
+        <img src="/assets/partners/partner6.png" alt="partners">
+        <img src="/assets/partners/partner7.png" alt="partners">
+        <img src="/assets/partners/partner8.png" alt="partners">
+      </div>
+    </section>
```

Comment:
Please the mobile view of the homepage in the design given doesnt include the `partners` section, it is only included in the about page. Please can you make this correction 😄 

Design given :
![image](https://github.com/Fombi-Favour/microverse-capstone-1/assets/20537712/74b8a61a-8290-4532-85bf-13a6d7c49d1c)

Your design :
![image](https://github.com/Fombi-Favour/microverse-capstone-1/assets/20537712/5be99bd2-d1cf-4cd5-957c-04f07a569701)